ytmp3/downloader.py

```diff
@@ -36,29 +33,34 @@ def __init__(self, bot, logging, options):
     async def ping(self, ctx):
         if not ctx.guild:
             return
-
         self.__log(f"User {ctx.author.id} pinged inside guild {ctx.guild.id}")
 
         await ctx.reply("Pong")
 
     @commands.command()
     async def download(self, ctx, url):
-        """
-        Downloads video from provided URL,
-        reply directly to discord user with URL of converted .mp3 file
-        """
         # Don't serve via direct message
         if not ctx.guild:
             return
 
         self.__log(f"User {ctx.author.id} attempted to download {url} inside guild ID {ctx.guild.id}")
 
         # Check URL for nasties like &list=
-        containsList = not not re.search("list=", url)
+        containsList = bool('list=' in url)
```

Comment:
Remove the `bool()`, the `in` operator already returns a boolean so there's no need to convert it to `bool()`